HighAvailabilityModule.E2ETest.TestCases/Infrastructure/TestClient.cs

```diff
@@ -50,7 +49,7 @@ public async Task<HeartBeatEntry> GetHeartBeatEntryAsync(string utype)
         {
             await this.LoseMessage();
```

Comment:
Should this line also be changed to await this.LoseMessage().ConfigureAwait(false); ?

Reply:
Yes. Thanks.